src/libs/auth.js

```diff
@@ -53,6 +53,10 @@ const isGoogleAuthority = () => {
   return _.startsWith('https://accounts.google.com', authStore.get().oidcConfig.authorityEndpoint)
 }
 
+export const isAzureIdp = () => {
```

Comment:
We already have an isAzureUser method-- can we standardize on a single one?

https://github.com/DataBiosphere/terra-ui/blob/41cf6a3a27a133a397a2238c6b967fa0a47e709e/src/libs/auth.js#L191

Reply:
I missed this, thanks! I'll standardize -- looks like the current `isAzureUser` method re-decodes the JWT to get the `idp` claim. Think we can just store `idp` in the profile upon initial decode.

Reply:
Done!